mobile_back_apple/cpp/backend_coreml/coreml_settings.pbtxt

```diff
@@ -1,58 +1,167 @@
 # proto-file: flutter/cpp/proto/backend_setting.proto
 # proto-message: BackendSetting
 
-
 benchmark_setting {
   benchmark_id: "image_classification"
-  accelerator: "ane"
-  accelerator_desc: "Apple Neural Engine"
   framework: "Core ML"
-  model_path: "https://github.com/mlcommons/mobile_models/raw/main/v3_0/CoreML/MobilenetEdgeTPU.mlmodel"
-  model_checksum: "39483b20b878d46144ab4cfe9a3e5600"
+  delegate_choice: {
+    delegate_name: "All"
+    accelerator_name: "all"
+    accelerator_desc: "All compute units"
+    model_path: "https://github.com/mlcommons/mobile_models/raw/main/v3_0/CoreML/MobilenetEdgeTPU.mlmodel"
+    model_checksum: "39483b20b878d46144ab4cfe9a3e5600"
+  }
+  delegate_choice: {
+    delegate_name: "CPU & GPU"
+    accelerator_name: "cpu&gpu"
+    accelerator_desc: "CPU and GPU"
+    model_path: "https://github.com/mlcommons/mobile_models/raw/main/v3_0/CoreML/MobilenetEdgeTPU.mlmodel"
+    model_checksum: "39483b20b878d46144ab4cfe9a3e5600"
+  }
+   delegate_choice: {
+    delegate_name: "CPU & NE"
+    accelerator_name: "cpu&ne"
+    accelerator_desc: "CPU and Neural Engine"
```

Comment:
It's more common to write Apple Neural Engine or ANE. So I would rename "cpu&ne" to "cpu&ane".